extensions/server-report/package.json

```diff
@@ -0,0 +1,395 @@
+{
+	"name": "server-report",
+	"displayName": "Server Reports",
```

Comment:
Would be nice if you went through and localized this once you get the initial checkin done